samples/sensor/icm42670S/aml_pointing/arduino_i2c.overlay

```diff
@@ -16,6 +16,12 @@
 		compatible = "invensense,icm42670S";
 		reg = <0x68>;
 		int-gpios = <&arduino_header 8 GPIO_ACTIVE_HIGH>; /* D2 */
+		
+		power-mode = "low noise";
+		accel-hz = <100>;
+		accel-fs = <16>;
+		gyro-hz = <100>;
+		gyro-fs = <2000>;
```

Comment:
I thought it was easy to have these values redefined in overlay because it's more accessible. But it's not needed. But the configuration is needed to have the AML algorithm properly running.

Reply:
Good to add this in the README instead.